src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -56,6 +55,7 @@
 import checkers.inference.model.Slot;
 import checkers.inference.model.VariableSlot;
 import checkers.inference.qual.VarAnnot;
+import checkers.inference.typearginference.InferenceTypeArgumentInference;
```

Comment:
Oh, I forgot to remove this import after playing with `InferenceTypeArgInference` in inferATF. Since this PR is already merged, I will remove this in another small PR.